### DISCUSSION TITLE
Activate scene from physical device

### DISCUSSION THREAD

**Discussion by alexojegu** [zwave-js/zwave-js-ui#3910] [Q&A] [answered]
I have two scenes created and one device with `Central Scene v3` capabilities. How can I activate a scene from the device?
 - Scene 1: Activate on `Central Scene v3` -> `Scene 001`
 - Scene 2: Activate on `Central Scene v3` -> `Scene 002`

**Comment by kpine** [comment]
You would press one of the buttons on the device, 1x, 2x, hold, etc.

**Reply by alexojegu** [reply]
I have not explained myself well.

I can't find in the Z-Wave JS UI where to configure which scene is activated when pressing one of the device's buttons (1x, 2x, hold). Is it possible to do this directly in Z-Wave JS UI or do I need to use, for example, Home Assistant?

**Reply by kpine** [reply]
You don't, the scene ID is dictated by the design of the device. For example, for on on/off switch, double tapping the top button could be Scene 1 and double tapping the button could be Scene 2. For a 5 button controller like a Zooz ZEN32, each button is a scene.

I suppose it's possible some device could allow you to configure scene IDs via configuration parameters (I can't recall any off the top of my head), but generally it's a static value.

**Reply by kpine** [reply]
Are you talking about the scenes in the ZUI GUI?

![image](https://github.com/user-attachments/assets/3274eb3a-895d-4144-a06d-445e5bc325d9)

**Reply by alexojegu** [reply]
I have this:
![image](https://github.com/user-attachments/assets/13a43eb4-f232-4093-90db-92a431e8b039)

And this:
![image](https://github.com/user-attachments/assets/5f658dbe-8152-44d8-b2b6-a109cee1a487)

So I just need to rename the scene from "Persianas Bajar" to "Scene 001"?

**Reply by alexojegu** [reply]
> Are you talking about the scenes in the ZUI GUI?

Yeah, that is.

**Reply by kpine** [reply, marked as answer]
The second picture are Z-Wave JS UI scenes. Those are activated via MQTT. Nothing to do with device Central Scene. You should use your automation software to trigger things on Central Scene events. If you are using HA it already supports such functionality.

[Main features](https://zwave-js.github.io/zwave-js-ui/#/?id=main-features):
> Scene Management: Create scenes and **trigger them by using MQTT apis** (with timeout support)

https://zwave-js.github.io/zwave-js-ui/#/guide/mqtt?id=execute-scene

**Reply by alexojegu** [reply]
Ok. It can't be done directly in the Zwave JS UI. I need to use Home Assistant (or MQTT).

I understand that you were initially referring to "direct associations" (using "Node groups"). I tried that too, but they seem much more limited.

PS: Sorry if I'm not calling things by the right name, I'm not familiar with Z-Wave.

**Reply by kpine** [reply]
No, I wasn't referring to associations. I didn't understand at first you were referring to ZUI Scenes since you mentioned Central Scene. Central Scene is meant for controlling devices via your "hub", not direct control of devices. Associations are meant for direct control, and yes they are more limited to simple operations.
